ci-operator/step-registry/acm/tests/clc-create/README.md

```diff
@@ -0,0 +1,44 @@
+# acm-tests-clc-create-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+
+## Purpose
+
+To test the managed cluster creation feature of ACM.
+
+> **IMPORTANT**
+> You must use the [acm-tests-clc-destroy-ref](../clc-destroy/README.md) as a post step when using this clc-create-ref. If you do not and succeed in running the clc-create-ref then you will leave clusters running on the ACM QE team's cloud.
+
+>Example usage in a config file
+>```
+>    post:
+>    - ref: acm-tests-clc-destroy
+>    test:
+>    - ref: install-operators
+>    - ref: acm-mch
+>    - ref: acm-tests-clc-create
+>```
+
+## Process
+
+- Copies a secret file needed for test into env.
+- Sets three dynamic variables based on the ephemeral hub cluster that needs to be provisioned prior to running this.
+- Runs a [script from product QE's repo](https://github.com/stolostron/clc-ui-e2e/blob/main/execute_clc_interop_commands.sh) that does some additional config and ultimately runs the  script ([start-tests.sh](https://github.com/stolostron/clc-ui-e2e/blob/main/start-tests.sh)) to kick off a cypress test where we install managed-clusters.
+
+## Requirements
+
+
+### Infrastructure
+
+- A provisioned test cluster to target (hub).
```

Comment:
Maybe more accurate: "An existing OpenShift cluster to act as the target Hub to deploy managed clusters onto."